pkg/ocm/ocm.go

```
@@ -331,18 +337,21 @@ func (c *SdkClient) GetClusterHypershiftConfig(cluster *cmv1.Cluster) (*cmv1.Hyp
 	return resp.Body(), nil
 }
 
-func CheckIfUserBanned(ocmClient Client, cluster *cmv1.Cluster) (bool, string, error) {
+func CheckIfUserBanned(ocmClient Client, cluster *cmv1.Cluster) error {
 	user, err := GetCreatorFromCluster(ocmClient.GetConnection(), cluster)
 	if err != nil {
-		return false, "encountered an issue when checking if the cluster owner is banned. Please investigate.", err
+		return fmt.Errorf("while checking if the cluster owner is banned: %v", err)
```

golangci-lint (2.11.4)

[error] non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)